### ISSUE TITLE
where did the service overrides functionality move to?

### ISSUE THREAD

**Issue by acranox** [nathwill/chef-systemd#110]
In version 2.1.3 of the cookbook, a systemd service had an option for overrides that reset the variable from any prior assignments.  (see comment below from the README)
it is unclear how to replicate this functionality in the 3.0.x cookbook.

```
overrides | drop-in unit options that require a reset (e.g. "ExecStart" -> "ExecStart=" at top of section)
```

**Comment by nathwill** [comment]
@acranox the overrides were moved to a separate resource `systemd_service_drop_in`. there's a mapping of the override resources [here](https://github.com/nathwill/chef-systemd#units--unit-drop-ins), and the specifics for the [service unit type drop-ins](https://github.com/nathwill/chef-systemd#systemd_service_drop_in).

hope that helps, let me know if you have other questions!

**Comment by acranox** [comment]
What I need is output that looks like the one below.
The line with the empty assignment is necessary.  I'm not seeing how to do this with the new cookbook.  I've tried a few things with no luck.  If it's possible, maybe it could just be more clearly documented in case I'm not the only clueless person out there.

```
[Service]
ExecStart=
ExecStart=/foo/bar.sh
```

**Comment by nathwill** [comment]
oooh, `overrides` vs `override`... sorry, i misunderstood.

i believe this may have been mistakenly dropped during the rewrite... as an interim solution, systemd loads drop-ins in lexical order, so a reset can be achieved by using 2 resources and naming the reset drop-in resource such that it's loaded first. i'll take a look at adding this back in for a 3.1.0 ASAP.

i'd also like to rename the property to `resets` while i'm at it, to avoid the `override` & `overrides` confusion... any objection?

**Comment by acranox** [comment]
I'm glad to hear it wasn't my lack of skills that was preventing me from getting it to work.
I don't care what you call it.  I have to re-work my cookbooks to use this new version, so I won't be reusing code anyhow.
'reset' seems consistent with what the [systemd docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html) describe: `If the empty string is assigned to this option, the list of commands to start is reset, prior assignments of this option will have no effect.`

Thanks.  I'll skip the interim solution and keep using 2.1.x for the cookbook that I need this functionality in, and  I'll be looking for the release of 3.1.0.

**Comment by nathwill** [comment]
@acranox shipped a solution in 3.1.0, holler if that doesn't work for ya.
